module-1.md

```diff
@@ -0,0 +1,35 @@
+# 1: What is mesh?
+
+Total length:  1.5 hour
+Planned:      1.25 hour
+
+| **Section**                                        | **Length** |
+|----------------------------------------------------|------------|
+| **1: Topic introduction**                          | 15 min     |
+| **2: Hands-on and material exploration**           | 40 min     |
+| **3: Discussion and reflection activity**          | 20 min     |
+| **4: Wrap-up and week ahead**                      | 5 min      |
+
+## 1: Topic introduction
+
+- Discuss familiar topics like net neutrality, internet service providers, and internet censorship
```

Comment:
Should link to materials somewhere (incl. slide deck/site?), these could be contained within this repo